### PR TITLE
Update documentation and fix broken links

### PR DESCRIPTION
**Pull request by dikel**
**Description**:
- updates the java examples
- fixes broken links
- adds missing methods and properties to some of the files

**Fixes:** Java SDK's [Issue #1348](https://github.com/hashgraph/hedera-sdk-java/issues/1348)

### CODE REVIEW

**Review by ochikov — APPROVED** [review]

File: reference/core/Client.md [hunk 1]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
-**Read-only**. Use `setOperator` or `setOperatorWith` to set.
+Current LedgerId of the network; corresponds to ledger ID in entity ID checksum calculations.
 
 ---
 
-##### `operatorPublicKey`: [`?PublicKey`](reference/cryptography/PublicKey.md)
+##### `maxAttempts` () : `Uint32`
 
-**Read-only**. Use `setOperator` or `setOperatorWith` to set.
+Max number of attempts a request executed with this client will do.
```
Comment (SimiHunjan) [inline reference/core/Client.md:271]: Are there defaults set in the SDK?

File: reference/core/Client.md [hunk 2]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
 
 ---
 
-##### `requestTimeout`: `Duration`
+##### `maxBackoff` () : `Duration`
+
+The maximum amount of time to wait between retries
```
Comment (SimiHunjan) [inline reference/core/Client.md:276]: Are there defaults set in the SDK?

File: reference/core/Client.md [hunk 3]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
 
 ---
 
-##### `transportSecurity` () : `bool`
+##### `maxNodeAttempts` () : `Uint32`
 
-Enable or disable TLS for both networks.
+Max number of times any node in the network can receive a bad gRPC status before being removed from the network.
```
Comment (SimiHunjan) [inline reference/core/Client.md:282]: Are there defaults set in the SDK?

File: reference/core/Client.md [hunk 4]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
 ---
 
-##### `verifyCertificates` () : `bool`
+##### `minBackoff` () : `Duration`
 
-Enable or disable TLS certificate verification
+The minimum amount of time to wait between retries
 
-Only available in Java and Go SDKs
+---
```
Comment (SimiHunjan) [inline reference/core/Client.md:290]: Are there defaults set in the SDK?

File: reference/core/Client.md [hunk 5]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
+The list of network records
 
-Only available in Java and Go SDKs
+---
+
+##### `networkUpdatePeriod` : `Duration`
+
+If present, the client will periodically attempt to update its consensus node network in the background using
+an [`AddressBookQuery`](../network/AddressBookQuery.md) against its current mirror network.
+If the query fails, the consensus node network will remain unchanged and the query failure will be logged.
```
Comment (SimiHunjan) [inline reference/core/Client.md:308]: Are there defaults set in the SDK?

File: reference/core/Client.md [hunk 6]
```diff
@@ -217,88 +199,136 @@ instead the network will de-prioritize it.
...
-locked if set manually or if certain methods are called.
+##### `operatorPublicKey`: [`PublicKey?`](../cryptography/PublicKey.md)
+
+The key of the operator
+
+---
+
+##### `requestTimeout`: `Duration`
+
+Maximum amount of time a request can run
```
Comment (SimiHunjan) [inline reference/core/Client.md:326]: Are there defaults set in the SDK?

File: reference/core/Query.md [hunk 7]
```diff
@@ -1,44 +1,68 @@
...
 
 ### Properties
 
-##### `nodeAccountIds`: [`AccountId[]`](reference/cryptocurrency/AccountId.md)
+##### `maxAttempts`: `Uint32`
 
-The nodeaccount IDs that will be queried.
+The number of times to retry submitting this transaction. Transactions are retried when Hedera
+Hashgraph returns a status code that implies the transaction should be resubmitted, or when a node
+doesn't respond.
```
Comment (SimiHunjan) [inline reference/core/Query.md:28]: Are there defaults set in the SDK?

File: reference/core/Query.md [hunk 8]
```diff
@@ -1,44 +1,68 @@
...
 
 Defaults to `maxQueryPayment` from the `client`.
 
 ---
 
-##### **Write-only** `queryPayment`: [`?Hbar`](reference/Hbar.md)
+##### `minBackoff`: `Duration`
+
+The minimum amount of time to wait between retries
+
```
Comment (SimiHunjan) [inline reference/core/Query.md:48]: Are there defaults set in the SDK?

File: reference/core/Transaction.md [hunk 9]
```diff
@@ -39,82 +68,95 @@ the same `Transaction`
...
+### Properties
 
-**NOTE**: Freezing is required for certain operations such as signing and generating hash.
+##### **Read-Only** `defaultMaxTransactionFee`: [`Hbar`](../Hbar.md)
 
----
+Тhe maximum transaction fee the client is willing to pay
 
-##### `freezeWith` ( `client`: [`Client`](reference/core/Client.md) ): `Transaction`
+This value is set by the SDK
```
Comment (SimiHunjan) [inline reference/core/Transaction.md:128]: We should add the default value set in the SDK which is 1 I believe

**Review by SimiHunjan — APPROVED** [review]

**Pull request merged**